solutions/client-landing-zone/client-folder/standard/applications-infrastructure/host-project/network/psc/google-apis/dns.yaml

```diff
@@ -0,0 +1,129 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+# Private DNS zone for googleapis.com
+# SC-22
+apiVersion: dns.cnrm.cloud.google.com/v1beta1
+kind: DNSManagedZone
+metadata:
+  name: host-project-id-internet-googleapis-dns # kpt-set: ${host-project-id}-internet-googleapis-dns
```

Comment:
internet here is referencing the name of the vpc
you will have to do a find and replace for the word internet and change it to standard in every file of this package and potentially in the project-setup.

Reply:
thank you